hedyorg/hedy
# 💻 Add customize class button to teacher page

Fixes #5121

**How to test**

Follow these steps to verify this PR works as intended:

* Go to teacher page
* click customize class icon and check it sends you directly to the customize class page

**Checklist**
Done? Check if you have it all in place using this list: (mark with x if done)

- [x ] Contains one of the PR categories in the name
- [ x] Describes changes in the format above
- [ x] Links to an existing issue or discussion
- [ x] Has a "How to test" section

If you're unsure about any of these, don't hesitate to ask. We're here to help!
